lib/pluginmanager/pack.rb

```diff
@@ -0,0 +1,60 @@
+require "bootstrap/util/compress"
+require "fileutils"
+
+class LogStash::PluginManager::Pack < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => !LogStash::Environment.windows?
+  option "--zip", :flag, "compress package as a zip file", :default => LogStash::Environment.windows?
+  option "--[no-]clean", :flag, "clean up the generated dump of plugins", :default => true
+  option "--force", :flag, "Overwrite a previously generated package file", :default => false
```

Comment:
Both the `pack.rb` and the `unpack.rb` are sharing the same options, we could create a common class with that and the two commands could inherit it.